doc/plugins/eu.esdihumboldt.hale.doc.user/html/new/5_1_0.xhtml

```diff
@@ -3,16 +3,29 @@
 <html xmlns="http://www.w3.org/1999/xhtml">
 <head>
 <link rel="stylesheet" type="text/css" href="new.css" />
-<title>What's new in hale»studio 5.0.1?</title>
+<title>What's new in hale»studio 5.1.0?</title>
 </head>
 <body>
 	<h1>What's new in hale»studio 5.1.0?</h1>
 
 	<ul>
-		<li>Update Map using OpenStreetMap</li>
-		<li>Consistent Experience for Excel and CSV</li>
-		<li>Better Excel Support</li>
-		<li>Enhanced Date Formatting for Excel and CSV</li>
+		<li><b>Better Excel Support:</b> You can now work with multiple sheets in Excel files all at once, making it easier to manage your data.</li>
```

Comment:
I would prefer it we keep the list relatively small with the most notable changes and in a concise way.

We still have the link to the complete changelog for more information.
The descriptions here are now more detailed than the changelog and some of them seem to represent an old state from the other PR where had removed or changed some of the items.

Reply:
> The descriptions here are now more detailed than the changelog and some of them seem to represent an old state from the other PR where had removed or changed some of the items.
any specific item where you see this? the list is exactly as in https://github.com/halestudio/hale/releases
in the draft release notes. I hope you can see it.

Reply:
Some examples:

> GeoJSON Geometry: We now prioritize properties with short paths as GeoJSON geometry property for easier access.
> Expanded JSON data import: Support for importing data from JSON files, enhancing the versatility of data import options.

This was a fix to the Json reader, that is a new feature that was not present in the previous release, so fixes that were done to the implementation before it was released should not be mentioned, just the feature.
See [conversation from previous PR](https://github.com/halestudio/hale/pull/1099#discussion_r1475714930).

> Schema Loading: Support extracting from JSON schema, providing users with greater flexibility in handling JSON data structures

This is misleading as we do not support JSON Schema.
See [conversation from previous PR](https://github.com/halestudio/hale/pull/1099#discussion_r1477800562).

> Enhanced Precision: Support up to 20 decimal places for source data geometries, ensuring heightened precision and accuracy in the exported data. This enhancement addresses the demand for meticulous spatial data representation, empowering users with more precise and reliable results.

This is misleading, see [conversation from previous PR](https://github.com/halestudio/hale/pull/1099#discussion_r1477807662).